pkg/registry/flowcontrol/ensurer/strategy.go

```diff
@@ -49,9 +49,6 @@ type ensureStrategy interface {
 	// This comes handy in logging.
 	Name() string
 
-	// ShouldCreate returns true if a missing configuration object should be recreated.
```

Comment:
The comment on this function becomes wrong with this change, still saying that the operator has the option to delete these objects.

The function comment has a long-standing mis-direction.  The point is not "ensure that the P&F system itself won't crash", the controller takes care of itself even if there are no configuration objects at all.  The point of the mandatory config objects is to ensure actual existing config objects that are consistent with the non-optional behavior.